lib/modules/manager/mix/artifacts.ts

```diff
@@ -11,8 +11,8 @@ import {
 } from '../../../util/fs';
 import * as hostRules from '../../../util/host-rules';
 import { regEx } from '../../../util/regex';
-
 import type { UpdateArtifact, UpdateArtifactsResult } from '../types';
+import { aliasRecordToRepositories } from './utils';
```

Comment:
split datasource and manager changes to separate PR

Reply:
#29775